deploy/olm-catalog/ibm-block-csi-operator/1.7.0/manifests/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -111,7 +111,7 @@ metadata:
         }
       ]
 spec:
-  displayName: "Operator for IBM block storage CSI driver"
+  displayName: "IBM block storage CSI driver plugin"
```

Comment:
hould be "IBM block storage CSI driver operator"

Reply:
done